ca/ca.go

```diff
@@ -528,6 +531,51 @@ func (ca *CA) getTLSConfig(auth *authority.Authority) (*tls.Config, error) {
 	return tlsConfig, nil
 }
 
+// getTLSClientConfig returns a TLSConfig for the CA with a self-renewing client
+// certificate.
+func (ca *CA) getTLSClientConfig(auth *authority.Authority) (*tls.Config, error) {
```

Comment:
Mostly copied from https://github.com/smallstep/certificates/pull/1001/commits/a30444b0d4c3f0af514b428f59d2989cf9b5b358#diff-9033031dd64a1f3c17bc6f97c5e0c3a0375eafa6e81bbbbbaccedc6446d94c8dR471

Reply:
We should add this back, passing the server tlsConfig or the renewer.

Reply:
627ee201aaef760bbcf7a1ff26daf6caae9ed34a